projects/1.20/assets/electrodynamics/electrodynamics/lang/zh_cn.json

```
@@ -0,0 +1,381 @@
+{
+    "item.electrodynamics.guidebook": "EEC 第一版",
+    "item.electrodynamics.laminatedcoil": "叠层铜线圈",
+    "item.electrodynamics.mechanicalvalve": "机械阀门",
+    "item.electrodynamics.plasticfibers": "塑料纤维",
+    "item.electrodynamics.fiberglasssheet": "玻璃纤维板",
+    "item.electrodynamics.platecopper": "铜板",
+    "item.electrodynamics.portablecylinder": "便携式气瓶",
+    "item.electrodynamics.pressuregauge": "压力表",
+    "item.electrodynamics.raworechromium": "粗铬铁矿",
+    "item.electrodynamics.raworefluorite": "氟石矿",
+    "item.electrodynamics.raworelepidolite": "粗锂矿",
+    "item.electrodynamics.raworethorium": "粗钍矿",
+    "item.electrodynamics.raworetitanium": "粗金红石矿",
+    "item.electrodynamics.raworeuranium": "粗铀矿",
+    "item.electrodynamics.raworevanadinite": "粗钒矿",
+
+    "block.electrodynamics.orechromium": "铬铁矿石",
```

```suggestion
    "block.electrodynamics.orechromium": "铬铁矿矿石",
```
术语：Chromite → 铬铁矿 Ore → 矿石
下方矿石名称一并修改